webapp/components/sidebar_header_dropdown.jsx

```diff
@@ -41,6 +41,7 @@ export default class SidebarHeaderDropdown extends React.Component {
         this.handleAboutModal = this.handleAboutModal.bind(this);
         this.aboutModalDismissed = this.aboutModalDismissed.bind(this);
         this.toggleAccountSettingsModal = this.toggleAccountSettingsModal.bind(this);
+        this.switchTeam = this.switchTeam.bind(this);
```

Comment:
Well once the team sidebar is in, we might need to remove the switching links from the menu as well, so we can remove that from there as well.